content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -30,34 +30,62 @@ Using the latest version of kubectl helps avoid unforeseen issues.
 
 ### Install kubectl binary with curl on Linux
 
-1. Download the latest release with the command:
+1. Download the latest stable release with the command:
 
+    ```bash
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
     ```
-    curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
-    ```
 
-    To download a specific version, replace the `$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)` portion of the command with the specific version.
+    {{< note >}}
+To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
+
+For example, to download version {{< param "fullversion" >}} on Linux, type:
+
+```bash
+curl -LO https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/linux/amd64/kubectl
+```
+    {{< /note >}}
+
+    **Validate the binary (optional)**
+
+    Checksum files are available to validate downloaded binaries.
 
-    For example, to download version {{< param "fullversion" >}} on Linux, type:
-    
+    To download the checksum file:
+    ```bash
+    curl -LO "https://dl.k8s.io/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
     ```
-    curl -LO https://storage.googleapis.com/kubernetes-release/release/{{< param "fullversion" >}}/bin/linux/amd64/kubectl
+
+    Validate the binary against the checksum file:
+    ```bash
+    echo "$(<kubectl.sha256) kubectl" | sha256sum --check
     ```
 
-2. Make the kubectl binary executable.
+    This should result in: `kubectl: OK`
 
+    If the check fails, `sha256` will report something like:
+    ```bash
+    kubectl: FAILED
+    sha256sum: WARNING: 1 computed checksum did NOT match
     ```
+
+    {{< caution >}}
+    Be sure to download the same version of the binary and checksum.
+    {{< /caution >}}
+
+1. Make the kubectl binary executable.
+
+    ```bash
     chmod +x ./kubectl
     ```
 
-3. Move the binary in to your PATH.
+1. Move the binary in to your PATH.
 
-    ```
+    ```bash
     sudo mv ./kubectl /usr/local/bin/kubectl
```

Comment:
Multiuser setups with `kubectl` installed must be rare; even so, let's explain how to provide a trusted `kubectl`:
```suggestion
    sudo mv ./kubectl /usr/local/bin/kubectl && \
    sudo chown root: /usr/local/bin/kubectl
```

For Linux-specific instructions, we could instead recommend:
```bash
sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
```
For people who haven't got root access, a common approach is to:
```bash
mkdir -p ~/.local/bin/kubectl
mv ./kubectl ~/.local/bin/kubectl
# and then add ~/.local/bin/kubectl to $PATH
```

Reply:
So the install command could replace lines 84 + 85 completely? Then I can add the second snippet as an addendum for people without root access.

Reply:
I've taken a shot at this edit for Linux and macOS:
* https://deploy-preview-25259--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl/#install-kubectl-binary-with-curl-on-linux
* https://deploy-preview-25259--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl/#install-kubectl-binary-with-curl-on-macos

Reply:
I'm not sure if the `install` command works the same on macOS as on Linux, so I used your initial suggested change there.